src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIClientExtensions.cs

```diff
@@ -14,6 +16,26 @@ namespace Microsoft.Extensions.AI;
 /// <summary>Provides extension methods for working with <see cref="OpenAIClient"/>s.</summary>
 public static class OpenAIClientExtensions
 {
+    /// <summary>Key into AdditionalProperties used to store a strict option.</summary>
+    internal const string StrictKey = "strictJsonSchema";
+
+    /// <summary>Gets the default OpenAI endpoint.</summary>
+    internal static Uri DefaultOpenAIEndpoint { get; } = new("https://api.openai.com/v1");
+
+    /// <summary>Gets a <see cref="ChatRole"/> for "developer".</summary>
+    internal static ChatRole ChatRoleDeveloper { get; } = new ChatRole("developer");
+
+    /// <summary>
+    /// Gets the JSON schema transformer cache conforming to OpenAI <b>strict</b> restrictions per https://platform.openai.com/docs/guides/structured-outputs?api-mode=responses#supported-schemas.
+    /// </summary>
+    internal static AIJsonSchemaTransformCache StrictSchemaTransformCache { get; } = new(new()
```

Comment:
Can these be marked private now?

Reply:
It's still used directly by the places that use it with structured output / response format, where we always apply it.